charts/incubator/kubernetes-reflector/values.yaml

```diff
@@ -0,0 +1,80 @@
+image:
+  # TODO: update image to use TC repo
+  repository: emberstack/kubernetes-reflector
+  pullPolicy: IfNotPresent
+  tag: 7.1.210
+
+env:
+  ES_Serilog__MinimumLevel__Default: Information
+  ES_Reflector__Watcher__Timeout: ""
+  ES_Reflector__Kubernetes__SkipTlsVerify: "false"
+
+operator:
+  register: true
+
+portal:
+  open:
+    enabled: false
+
+rbac:
+  main:
+    enabled: true
+    primary: true
+    clusterWide: true
+    rules:
+    - apiGroups:
+      - ""
+      resources:
+      - "configmaps"
+      - "secrets"
+      verbs:
+      - "get"
+      - "list"
+      - "watch"
+      - "create"
+      - "update"
+      - "patch"
+      - "delete"
+    - apiGroups:
+      - ""
+      resources:
+      - "namespaces"
+      verbs:
+      - "watch"
+      - "list"
+
+serviceAccount:
+  main:
+    enabled: true
+    primary: true
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+
+podOptions:
+  automountServiceAccountToken: true
```

Comment:
Just a note, those can be set directly under `main` for container and under `podSpec` for pod options.
When set at the top level, applies to all pods/containers in this chart. 

Personally I prefer to use the more specific scope, but top level is also fine.
Just thought I should mention it, in case you haven't notice